pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1248 @@
+      - mysql-qa
+```
+Explanation for the above configuration:
+
+- `spec.backupConfig.target.name`: Tells `TVK` what target `name` to use for storing backups.
+- `spec.backupConfig.target.namespace`: Tells `TVK` in what namespace the target was created.
+- `spec.backupComponents`: Defines a `list` of `resources` to back up (can be `namespaces` or `Helm releases`).
+
+Typical `Backup` CRD looks like below:
+
+```yaml
+apiVersion: triliovault.trilio.io/v1
+kind: Backup
+metadata:
+  name: mysql-qa-helm-release-full-backup
+  namespace: demo-backup-ns
+spec:
+  type: Full
+  backupPlan:
+    name: mysql-qa-helm-release-backup-plan
+    namespace: demo-backup-ns
+```
+Explanation for the above configuration:
+
+- `spec.type`: Specifies backup type (e.g. `Full` or `Incremental`).
+- `spec.backupPlan`: Specifies the `BackupPlan` which this `Backup` should use.
+
+Steps to initiate the `mysql-qa` Helm release one time backup:
+<ol>
+  <li>First, make sure that the `mysql-qa` is deployed in your cluster by following steps  from (pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md#step-4---Creating the mysql-qa Helm Release Backup) section.</li>
+  </li>  
+  <li>Next, change directory where the `docs` Git repository was cloned on your local machine:
+
+  ```shell
+  cd docs/pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/
+  ```
+  </li>  
+  <li>Then, open and inspect the mysql-qa helm release `BackupPlan` and `Backup` manifest files provided in the `pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md` repository, using an editor of your choice (preferably with `YAML` lint support). You can use [VS Code](https://code.visualstudio.com) for example:
+
+  ```shell
+  cat assets/manifests/mysql-qa-helm-release-backup-plan.yaml
+  cat assets/manifests/mysql-qa-helm-release-backup.yaml
+  ```
+  </li>  
+  <li>Create the `BackupPlan` resource, using `kubectl`:
+
+  ```shell
+  kubectl apply -f assets/manifests/mysql-qa-helm-release-backup-plan.yaml -n demo-backup-ns
+  ```
+  Now, inspect the `BackupPlan` status (targeting the `mysql-qa` Helm release), using `kubectl`:
+
+  ```shell
+  kubectl get backupplan mysql-qa-helm-release-backup-plan -n demo-backup-ns
+  ```
+  The output looks similar to (notice the `STATUS` column value which should be set to `Available`):
+
+  ```text
+  NAME                                  TARGET			...   STATUS
+  mysql-qa-helm-release-backup-plan   trilio-ovh-s3-target	...   Available
+  ```
+  </li>  
+  <li>Finally, create a `Backup` resource, using `kubectl`:
+
+  ```shell
+  kubectl apply -f assets/manifests/mysql-qa-helm-release-backup.yaml -n demo-backup-ns
+  ```
+  Now, inspect the `Backup` status (targeting the `mysql-qa` Helm release), using `kubectl`:
+
+  ```shell
+  kubect get backup mysql-qa-helm-release-full-backup -n demo-backup-ns
```

Comment:
replace 
  kubect get backup mysql-qa-helm-release-full-backup -n demo-backup-ns
with
  kubectl get backup mysql-qa-helm-release-full-backup -n demo-backup-ns

Reply:
Thank you @raclepoulpe . Fixed this in a recent commit.